rewrite-groovy/src/test/java/org/openrewrite/groovy/tree/ImportTest.java

```diff
@@ -90,4 +101,21 @@ void staticImportAlias() {
           )
         );
     }
+
+    @Test
+    void duplicateImports() {
+        rewriteRun(
+          groovy(
+            """
+                    import static      java.util.Collections.*    ;     import               static      java.util.Collections.*
```

Comment:
Note: Adding a commented out static import line (like `// import static foo.*`) causes an NPE.

Reply:
O boy, always those pesky comments. I'll create a follow up PR this fix this!

Reply:
Done ⬇️